CRM/Upgrade/Incremental/sql/5.63.alpha1.mysql.tpl

```diff
@@ -14,3 +14,13 @@ VALUES
     ( {$domainID}, 'https://civicrm.org/help?src=iam',       '{ts escape="sql" skip="true"}Get Help{/ts}',   'Get Help',   NULL, 'AND', @adminHelplastID, '1', NULL, 2 );
 
 UPDATE IGNORE `civicrm_navigation` SET `name` = 'Register Your Site', `label` = '{ts escape="sql" skip="true"}Register Your Site{/ts}' WHERE `name` = 'Register your site';
+
+{if $multilingual}
+    {foreach from=$locales item=locale}
+        UPDATE IGNORE civicrm_uf_field SET label_{$locale} = '{ts escape="sql"}Contribution Date{/ts}' WHERE (field_name = 'receive_date' AND label_{$locale} = '{ts escape="sql"}Received{/ts}' AND uf_group_id IN (10,11));
```

Comment:
I apologize since I never actually ran it, but now that I have and seeing that it doesn't work, I'm not sure this exact type of upgrade has ever been done before. All the previous upgrades against multilingual were schema updates, or were setting the value to `''`.

I can't see how this type of update would ever work because `{ts}` will **ALWAYS** be, say, english if I run the upgrade while logged in and running the upgrade in english. There's nothing here that switches the locale so that ts runs in the locale.

I appreciate the time you've put in but I'm going to suggest just skipping this update. Given how long the current wording has been the wording, people can always manually update their profiles if they really want different wording.

Reply:
Great @demeritcowboy, I agree with your position. in that case we can close this PR

Reply:
Sorry to have led you astray, @olayiwola-compucorp!